### PR TITLE
Style: AddCommentForm changed

### PR DESCRIPTION
**Pull request by Saifmagouri**
## Changes proposed

Improved the styling of the `AddCommentForm` component to enhance usability and user-friendliness.

## Screenshots
Capture d'écran 2024-04-28 204135

Capture d'écran 2024-04-28 204258
Capture d'écran 2024-04-28 204251

## Code of Conduct

- [x] By submitting this pull request, I confirm I've read and complied with the [CoC](https://github.com/thecyberworld/TheCyberHUB/blob/dev/CODE_OF_CONDUCT.md) 🖖

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

---

You can also join our [Discord](https://discord.gg/QHBPq6xP5p) community.
Feel free to check out other cool repositories of the [Thecyberworld](https://github.com/thecyberworld).
Join the Thecyberworld GitHub Organisation by raising an [issue](https://github.com/thecyberworld/Support/issues/new?assignees=&labels=invite+me+to+the+organisation&template=invitation.yml&title=Please+invite+me+to+the+GitHub+Community+Organization) (you will be sent an invitation).

### CODE REVIEW

**Comment by ArkadiK94** [comment]
@Saifmagouri you staged fileupload.md removal. Remove from git this change.

**Comment by ArkadiK94** [comment]
@Saifmagouri tell me if you need help with it

**Comment by Saifmagouri** [comment]
@ArkadiK94 fileupload.md was removed automatically because my antivirus was trigerred 
what should i do to complete the PR

**Comment by Saifmagouri** [comment]
@ArkadiK94 okay i will keep it in mind next time

File: src/components/Blogs/BlogComments/AddCommentForm.jsx [hunk 1]
```diff
@@ -35,20 +33,34 @@ const AddCommentForm = (props) => {
...
+                        right: "20px",
+                        backgroundColor: "red",
+                        color: "white",
+                        padding: "10px",
+                        borderRadius: "5px",
+                    }}
+                >
+                    {error}
+                </p>
+            )}
```
Comment (ArkadiK94) [inline src/components/Blogs/BlogComments/AddCommentForm.jsx:63]: @Saifmagouri nope delete it and use toast.warn method from "react-toastify" package

**Review by ArkadiK94 — CHANGES_REQUESTED** [review]
@Saifmagouri Thanks for your contribution, I have some comments.

https://github.com/thecyberworld/TheCyberHUB/assets/76536506/4b5a2d66-a8c1-4594-a001-3e6b3347567d

1. the style of the "Submit" button, do similar as in INTERNSHIP OPPORTUNITIES form

2. add border to the input box also as in INTERNSHIP OPPORTUNITIES form

3. also the border color of the comment box change it to be grey

File: src/components/Other/MixComponents/Buttons/ButtonElements.jsx [hunk 2]
```diff
@@ -98,38 +98,22 @@ export const RedirectButton = styled.a`
...
         background: #ff6b08;
         color: black;
         border-color: #343434;
-        // font-size: 18px;
-    }
-
-    @media screen and (max-width: 600px) {
-        padding: 10px 10px;
     }
 `;
```
Comment (ArkadiK94) [inline src/components/Other/MixComponents/Buttons/ButtonElements.jsx:119]: @Saifmagouri very problematic, you made a major changes without checking all the places where this button is used

image

For example:
Before the change:
image

After the change:
image
image

Next, time always if you want to change existed component make sure to check where it used 

Now:
1. create local component button style
2. adjacent the changes to the right ones in all places!

**Review by ArkadiK94 — CHANGES_REQUESTED** [review]
@Saifmagouri Hi, looks much better but please, fix the new comment.

**Comment by ArkadiK94** [comment]
@Saifmagouri hi, how is it going?

**Comment by ArkadiK94** [comment]
Because there is no response, I will modify it by myself...

**Comment by ArkadiK94** [comment]
@Saifmagouri Thank you very much for your contribution, I resolved the last comment.

**Review by ArkadiK94 — APPROVED** [review]

**Pull request merged**